website/static/css/video.css

```diff
@@ -0,0 +1,94 @@
+.babel-video {
+  border: 1px solid rgba(0, 0, 0, .2);
+  border-radius: .25rem;
+  height: 100%;
+  overflow: hidden;
+}
+
+.page-header {
```

Comment:
One issue I'm seeing is that you have styles that could be re-used across the other pages like in https://github.com/babel/website/pull/1530 - I would think video.css would only apply to that page (or that docusarus isolates each css but that's not the case)

Reply:
I initially felt that the style would be adjusted, so the style was not added to babel.css. I'm going to amend it.